elasticdl/python/tests/optimizer_wrapper_test.py

```diff
@@ -25,11 +25,11 @@
     load_module,
 )
 from elasticdl.python.elasticdl.layers.embedding import Embedding
-from elasticdl.python.master.optimizer_wrapper import OptimizerWrapper
 from elasticdl.python.ps.embedding_table import (
     EmbeddingTable,
     get_slot_table_name,
 )
+from elasticdl.python.ps.optimizer_wrapper import OptimizerWrapper
```

Comment:
How did we pass pre-commit CI before?

Reply:
I do not very understand your comment. Every PR has to pass pre-commit CI.

Reply:
Oops I thought this only changes the order of import. I just noticed it's a change from `master.xxx` to `ps.xxx`. Please ignore my comment.